src/content/docs/zh-cn/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,604 @@
+
+Astro v3.0 将 `inlineStylesheets` 的默认值更改为 `"auto"`。默认情况下，小于 `ViteConfig.build.assetsInlineLimit`（默认值：4kb）的样式表被内联。否则，项目样式将以外部样式表的形式发送。
+
+#### 我应该怎么做？
+如果你想保留你项目的当前行为，请将 `build.inlineStylesheets` 设置为以前的默认值 `"never"`：
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+	 build: {
+    inlineStylesheets: "never"
+  }
+})
+```
+
+### 默认值改变：图像服务
+
+在 Astro v2.x 中，Squoosh 是[默认的图像处理服务](/zh-cn/guides/images/#默认图像服务)。
+
+Astro v3.0 将 Sharp 作为默认的图像处理服务，并提供了一个配置选项来使用 Squoosh。
+
+#### 我应该怎么做？
+
+如果你希望继续使用 Squoosh 来转换你的图像，请使用以下配置更新你的配置：
+
+```ts title="astro.config.mjs" ins={4-6}
+import { defineConfig, squooshImageService } from "astro/config";
+
+export default defineConfig({
+  image: {
+    service: squooshImageService(),
+  }
+})
+```
+
+### 更新：HTTP 请求方法案例
+
+在 Astro v2.x 中，[HTTP 请求方法](/zh-cn/core-concepts/endpoints/#http-方法) 使用小写函数名称编写：`get`、`post`、`put`、`all` 和 `del`。
+
+Astro v3.0 使用大写函数名称，包括 `DELETE` 而不是 `del`。
+
+#### 我应该怎么做？
+
+重命名所有的函数为它们的大写等价物：
+
+- `get` to `GET`
+- `post` to `POST`
+- `put` to `PUT`
+- `all` to `ALL`
+- `del` to `DELETE`
+
+```js title="endpoint.ts" del={1} ins={2}
+export function get() {
+export function GET() {
+    return new Response(JSON.stringify({ "title": "Bob's blog" }));
+}
+```
+
+### 更新：多个 JSX 框架配置
+
+在 Astro v2.x 中，你可以在同一个项目中使用多个 JSX 框架集成（React、Solid、Preact）而不需要识别哪些文件属于哪个框架。
+
+Astro v3.0 现在要求你在有多个 JSX 框架集成时，使用新的 `include` 和 `exclude` 集成配置选项来指定你的文件使用哪个框架。这使得 Astro 更好地支持单框架使用，以及 React Fast Refresh 等高级功能。
+
+#### 我应该怎么做？
+
+如果你在同一个项目中使用多个 JSX 框架集成，请将 `include`（和可选的 `exclude`）设置为文件数组或文件夹。通配符可用于包含多个文件路径。
+
+我们建议将常见的框架组件放在同一个文件夹中（例如 `/components/react/` 和 `/components/solid/`），以便更容易地指定你的包含内容，但这不是必需的：
+
+```js ins={13,16,19}
+import { defineConfig } from 'astro/config';
+import preact from '@astrojs/preact';
+import react from '@astrojs/react';
+import svelte from '@astrojs/svelte';
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+
+export default defineConfig({
+  // 启用多个框架来支持所有不同类型的组件。
+  // 如果你只使用一个 JSX 框架，则不需要 `include`！
+  integrations: [
+    preact({
+      include: ['**/preact/*'],
+    }),
+    react({
+      include: ['**/react/*'],
+    }),
+    solid({
+      include: ['**/solid/*'],
+    }),
+  ],
+});
+```
+
+### 更新：`Astro.cookies.get(key)` 可以返回 `undefined`
+
+在 Astro v2.x 中，[`Astro.cookies.get(key)`](/zh-cn/reference/api-reference/#astrocookies) 总是会返回一个 [`AstroCookie` 对象](/zh-cn/reference/api-reference/#astrocookie)，即使 cookie 不存在。要检查它是否存在，你需要使用 `Astro.cookies.has(key)`。
+
+Astro v3.0 中，如果 cookie 不存在，`Astro.cookies.get(key)` 会返回 `undefined`。
+
+#### 我应该怎么做？
+
+这个更新不会破坏任何在使用 `Astro.cookies.get(key)` 之前检查 `Astro.cookie` 对象是否存在的代码，但现在不再需要。
+
+你可以安全的删除任何使用 `has()` 来检查 `Astro.cookies` 的值是否为 `undefined` 的代码：
```

Comment:
```suggestion
这个修改不会影响任何在调用 `Astro.cookies.get(key)` 之前用 `Astro.cookies.has(key)` 检查 cookie 是否存在的代码，但现在不再需要检查。

你可以安全的删除任何使用 `has()` 来检查 cookie 是否存在的代码：
```

The suggested change deviate a lot from the original English text. Waiting for feedback on the wording of the original text on Discord.